admin/build-doc

```diff
@@ -22,7 +22,11 @@ if command -v dpkg >/dev/null; then
 elif command -v yum >/dev/null; then
     for package in python-devel python-pip python-virtualenv doxygen ditaa ant libxml2-devel libxslt-devel Cython graphviz; do
 	if ! rpm -q $package >/dev/null ; then
-		missing="${missing:+$missing }$package"
+		command -v `echo $package | tr '[:upper:]' '[:lower:]' | awk -F "-" '{ if ($2 == "") print $1; else print $2}'`   > /dev/null;
```

Comment:
I am using FC25 and `ditaa` is available on my system. And that is the reason it is not giving any errors related to `ditaa` package. Errors are only related `python-virtualenv` and `Cython`.
```
$ cat /etc/redhat-release 
Fedora release 25 (Twenty Five)
$ rpm -q ditaa
ditaa-0.10-2.fc24.noarch
```

And after modifying the `admin/build-doc` with the changes in this PR, the `admin/build-doc` got triggered and completed successfully. 
```
$ admin/build-doc 
Top Level States:  ['RecoveryMachine']
Processing /home/ashishkumsingh/ceph/src/pybind/rados
  Requirement already satisfied (use --upgrade to upgrade): rados==2.0.0 from file:///home/ashishkumsingh/ceph/src/pybind/rados in ./virtualenv/lib/python2.7/site-packages
...
...

BUILD SUCCESSFUL
Total time: 5 seconds
```